--- .config/sxhkd/sxhkdrc.o	2020-03-25 19:17:29.260365362 +0000
+++ .config/sxhkd/sxhkdrc	2020-03-25 19:57:07.809535828 +0000
@@ -49,7 +49,7 @@
 super + shift + Escape
 	prompt 'Leave Xorg?' 'killall Xorg'
 super + x
-	ifinstalled slock && ( slock & xset dpms force off ; mpc pause ; pauseallmpv )
+	ifinstalled slock && ( slock & xset dpms force off ; lmc pause )
 XF86Launch1
 	xset dpms force off
 XF86TouchpadToggle
@@ -76,21 +76,21 @@

 # Pause audio
 super + p
-	mpc toggle
+	lmc toggle
 super + shift + p
-	mpc pause; pauseallmpv
+	lmc pause
 # Changes to next/previous tracks
 super + {comma,period}
-	mpc {prev,next}
+	lmc {prev,next}
 # Restart track
 super + shift + less
-	mpc seek 0%
+	lmc replay
 # Seek foward in song
 super + {_,shift +} bracketright
-	mpc seek +{10,120}
+	lmc seek +{10,120}
 # Seek backward in song
 super + {_,shift +} bracketleft
-	mpc seek -{10,120}
+	lmc seek -{10,120}
 # Increase volume
 super + {equal,plus}
 	amixer sset Master {5,15}%+; pkill -RTMIN+10 blocks
@@ -106,11 +106,11 @@
 XF86Audio{Raise,Lower}Volume
 	amixer sset Master 5%{+,-}; pkill -RTMIN+10 blocks
 XF86Audio{Next,Prev}
-	mpc {next,prev}
+	lmc {next,prev}
 XF86Audio{Pause,Play,Stop}
-	mpc {pause,play,stop}
+	lmc {pause,play,pause}
 XF86Audio{Rewind,Forward}
-	mpc seek {-,+}10
+	lmc seek {-,+}10
 XF86AudioRecord
 	dmenurecord
 XF86AudioMedia
@@ -127,7 +127,7 @@
 XF86DOS
 	$TERMINAL
 XF86ScreenSaver
-	mpc pause; pauseallmpv; i3lock -e -f -c 1d2021; xset dpms force off
+	lmc pause; i3lock -e -f -c 1d2021; xset dpms force off
 XF86TaskPane
 	$TERMINAL -e htop
 XF86Mail
