--- .config/sxhkd/sxhkdrc	2019-12-10 10:16:41.921230823 +0000
+++ .config/sxhkd/sxhkdrc	2019-12-10 10:21:46.954585232 +0000
@@ -51,7 +51,7 @@
 super + shift + Escape
 	prompt 'Leave Xorg?' 'killall Xorg'
 super + x
-	ifinstalled slock && ( slock & xset dpms force off ; mpc pause ; pauseallmpv )
+	ifinstalled slock && ( slock & xset dpms force off ; lmc pause ; pauseallmpv )
 XF86Launch1
 	xset dpms force off
 XF86TouchpadToggle
@@ -78,41 +78,41 @@

 # Pause audio
 super + p
-	mpc toggle
+	lmc toggle
 super + shift + p
-	mpc pause; pauseallmpv
+	lmc pause; pauseallmpv
 # Changes to next/previous tracks
 super + {comma,period}
-	mpc {prev,next}
+	lmc {prev,next}
 # Restart track
 super + shift + less
-	mpc seek 0%
+	lmc replay;
 # Seek foward in song
 super + {_,shift +} bracketright
-	mpc seek +{10,120}
+	lmc forward {10,120}
 # Seek backward in song
 super + {_,shift +} bracketleft
-	mpc seek -{10,120}
+	lmc back {10,120}
 # Increase volume
 super {_,shift +} + plus
-	amixer sset Master {5,15}%+; pkill -RTMIN+10 $STATUSBAR
+	lmc up {5, 15}
 # Decrease volume
 super {_,shift +} + minus
-	amixer sset Master {5,15}%-; pkill -RTMIN+10 $STATUSBAR
+	lmc down {5, 15}
 super + shift + m
-	amixer sset Master toggle; pkill -RTMIN+10 $STATUSBAR
+	lmc mute

 # Audiokeys
 XF86AudioMute
-	amixer sset Master toggle; pkill -RTMIN+10 $STATUSBAR
+	lmc mute
 XF86Audio{Raise,Lower}Volume
-	amixer sset Master 5%{+,-}; pkill -RTMIN+10 $STATUSBAR
+	lmc {up, down} 15
 XF86Audio{Next,Prev}
-	mpc {next,prev}
-XF86Audio{Pause,Play,Stop}
-	mpc {pause,play,stop}
+	lmc {next,prev}
+XF86AudioPlay
+	lmc toggle
 XF86Audio{Rewind,Forward}
-	mpc seek {-,+}10
+	lmc {back, forward} 10
 XF86AudioRecord
 	dmenurecord
 XF86AudioMedia
@@ -129,7 +129,7 @@
 XF86DOS
 	$TERMINAL
 XF86ScreenSaver
-	mpc pause; pauseallmpv; i3lock -e -f -c 1d2021; xset dpms force off
+	lmc pause; pauseallmpv; i3lock -e -f -c 1d2021; xset dpms force off
 XF86TaskPane
 	$TERMINAL -e htop
 XF86Mail
